modmesh/view.py

```diff
@@ -60,6 +62,30 @@ def _load():
     if enable:
         for name in _from_impl:
             globals()[name] = getattr(_vimpl, name)
+        # Try to find the PUI in thirdparty, if failed to find PUI
+        # modmesh will raise ImportError and terminate itself.
+        filename = os.path.join('thirdparty', 'PUI')
+        path = os.getcwd()
+        try:
+            while True:
+                if os.path.exists(os.path.join(path, filename)):
+                    break
+                if path == os.path.dirname(path):
+                    path = None
+                    break
+                else:
+                    path = os.path.dirname(path)
+            if path is None or not os.path.exists(os.path.join(path,
```

Comment:
Check if the `PUI` folder in `thirdparty` contains `PUI` folder or not, if the user does not initialize the submodule, then the `PUI` in `thirdparty` will be an empty folder.